src/components/cms-components/BlogHighlight/BlogHighlight.astro

```diff
@@ -50,7 +49,7 @@ const blogPost = Astro.props.blogPost.blogPost;
           {blogPost.intro?.intro && (
             <div class="text-sm font-light">
               <PortableText
-                components={portableBlogHighlightComponents}
+                components={defaultComponents}
```

Comment:
I think the styles for the bloghighlight are gone. These have different styles that's why I had created custom components for them.